src/SourceBuild/content/test/TestUtilities/ConditionalDiscovererException.cs

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Xunit;
```

Comment:
I don't think this is an appropriate namespace as this is not part of the Xunit product.  It can be confusing for folks when working with the tests.

Reply:
FWIW we have been using this namespace in some components for a while now, i.e. https://github.com/dotnet/arcade/blob/f54c598d137ab88bd473ea670b3c751be0540244/src/Microsoft.DotNet.XUnitExtensions/src/TestRuntimes.cs#L6 but I'm happy to change. Will send a commit to update this.

Reply:
Updated with the latest commit.